### PR TITLE
Number finder

### PR DESCRIPTION
**Pull request by Gaddsky**

### CODE REVIEW

File: gadd/src/main/java/textDictionary/CreateDictionary.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+public class CreateDictionary {
+    private String filePath;
+
+    public CreateDictionary(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public HashMap getResult() {
+        try {
+            ArrayList wordsList = getWordsFromFile();
```
Comment (pasby) [inline gadd/src/main/java/textDictionary/CreateDictionary.java:20]: please use interfaces in defining vars (List here) here and further

File: gadd/src/main/java/textDictionary/CreateDictionary.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            HashSet wordsSet = convertToSet(wordsList);
+            return countWords(wordsSet, wordsList);
+        } catch (IOException e) {
+            System.out.println(e);
```
Comment (pasby) [inline gadd/src/main/java/textDictionary/CreateDictionary.java:24]: you can use e.printStackTrace();

File: gadd/src/main/java/textDictionary/CreateDictionary.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        return null;
+    }
+
+    private ArrayList getWordsFromFile() throws IOException {
+        ArrayList wordsArray = new ArrayList();
+        FileReader fr = new FileReader(filePath);
+        BufferedReader textReader = new BufferedReader(fr);
+        String textLine;
+        while ((textLine = textReader.readLine()) != null) { // to the end of file
+            String[] wordsInLine = textLine.split(" "); // split string to words
```
Comment (pasby) [inline gadd/src/main/java/textDictionary/CreateDictionary.java:36]: what about dots or commas or other punctuation?

File: gadd/src/main/java/textDictionary/Run.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.HashMap;
+import java.util.Set;
+
+public class Run {
+    public static void main(String[] args) {
+        CreateDictionary cd = new CreateDictionary("/home/gadd/example_text.txt");
+        HashMap dict = cd.getResult();
+        Set dictKeys = dict.keySet();
+        for (Object key : dictKeys) {
```
Comment (pasby) [inline gadd/src/main/java/textDictionary/Run.java:11]: it is better to iterate by Entity. f.e. :
`Map<String, Integer> zu = new HashMap<>();
        for (Map.Entry<String, Integer> item : zu.entrySet()) {
            System.out.println(item.getKey()+" "+item.getValue());
        }`

File: gadd/src/main/java/textDictionary/CreateDictionary.java (hunk 3 above)
Comment (Gaddsky) [inline gadd/src/main/java/textDictionary/CreateDictionary.java:36]: All non-alphabetic symbols removes by function strip()

**Pull request closed without merging**